src/components/previewers/ImagePreviewer.jsx

```
@@ -24,6 +24,7 @@ const ImagePreviewer = ({
   description,
   userImage,
   userName,
+  isEmoji = false,
```

ESLint

[error] 'isEmoji' is missing in props validation

(react/prop-types)